development/build/scripts.js

```diff
@@ -1279,20 +1282,24 @@ function renderJavaScriptLoader({
     (label) => `./${label}.js`,
   );
 
-  const securityScripts = applyLavaMoat
-    ? ['./runtime-lavamoat.js', './lockdown-more.js', './policy-load.js']
-    : [
-        './lockdown-install.js',
-        './lockdown-run.js',
-        './lockdown-more.js',
-        './runtime-cjs.js',
-      ];
+  let securityScripts = [
+    './lockdown-install.js',
+    './lockdown-run.js',
+    './lockdown-more.js',
+    './runtime-cjs.js',
+  ];
+
+  if (applyLavaMoat) {
+    securityScripts = ['./lockdown-more.js', './policy-load.js'];
+    if (overrideLavamoatRuntime === false) {
+      securityScripts.unshift('./runtime-lavamoat.js');
+    }
+  }
 
   const requiredScripts = [
-    './snow.js',
-    './use-snow.js',
-    './sentry-install.js',
+    ...(overrideSnow ? ['./use-snow.js'] : ['./snow.js', './use-snow.js']),
```

Comment:
I then thought maybe we don't need to activate snow in this window, but we do. I experimented with multiple combinations and got new errors related to snow